main/src/MillMain.scala

```diff
@@ -71,6 +72,10 @@ case class MillConfig(
 object MillMain {
 
   def main(args: Array[String]): Unit = {
+
+    if (Properties.isWin && System.console() != null)
+      io.github.alexarchambault.windowsansi.WindowsAnsi.setup()
```

Comment:
Where does this packages comes from. Assuming it is a transitive dependency of coursier, we should make it explicit in mill's dependencies, too.

Is it ok to call the `setup()` method multiple times? Is there some _cleanup_ needed?

Reply:
Sure, I can add it explicitly. It's brought by `io.get-coursier::coursier-cache`.

`setup()` can be called multiple times, yes. It'll just return `false` starting from the second call. The only requirement is to only call it on Windows (JNI will complain else), when `System.console() != null` (or it'll throw an `IOException`).